megablocks/layers/glu.py

```diff
@@ -0,0 +1,75 @@
+from megablocks.layers import common
+from megablocks.layers import gelu
+from megablocks.layers.mlp import SparseMLP, create_dmoe_expert_weights
```

Comment:
for this PR, maybe I keep this import as is in this file, but in the future it seems like it would make stuff to refactor into a separate `mlp/` folder with a`utils.py`?

Reply:
SGTM!